lib/commands/help.js

```diff
@@ -16,20 +16,12 @@ const exampleEmbed = {
             value: 'checks if the bot is active',
         },
         {
-            name: 'role @role',
-            value: 'assigns @role to caller',
+            name: 'role role',
```

Comment:
Can we change this to `role name-of-role` to make it more clear for the user

Reply:
updated